RELEASE/scripts/autoscend/quests/level_10.ash

```diff
@@ -333,11 +333,13 @@ boolean L10_holeInTheSkyUnlock()
 	}
 	int day = get_property("shenInitiationDay").to_int();
 	boolean[location] shenLocs = shenSnakeLocations(day, 0);
-	if(!needStarKey() && !(shenLocs contains $location[The Hole in the Sky]))
+	if((!needStarKey() || (can_interact() && LX_getStarKey())) && 
```

Comment:
this is wrong. LX_getStarKey() can spend adventures.
it should not return false if it successfully adventures as this function does.
in general it should be
if(conditions)
{
      if(LX_getStarKey()) return true;
}